x/concentrated-liquidity/types/errors.go

```diff
@@ -0,0 +1,22 @@
+package types
+
+import (
+	"errors"
+	fmt "fmt"
+)
+
+// x/concentrated-liquidity module sentinel errors.
+type InvalidLowerUpperTickError struct {
+	LowerTick int64
+	UpperTick int64
+}
+
+func (e InvalidLowerUpperTickError) Error() string {
+	return fmt.Sprintf("Lower tick must be lesser than upper. Got lower: %d, upper: %d", e.LowerTick, e.UpperTick)
+}
+
+var (
+	ErrInvalidLowerTick         = errors.New("lower tick must be in valid range")
+	ErrLimitUpperTick           = errors.New("upper tick must be in valid range")
+	ErrNotPositiveRequireAmount = errors.New("required amount should be positive")
```

Comment:
Do you mind formatting the actual values and ranges here, too? I think knowing these would be useful since they are client facing

Reply:
yes sure